### ISSUE TITLE
[Druid] Druid turning.

### ISSUE THREAD

**Issue by jarred1119** [dalaranwow/dalaran-wow#4557]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: While keyboard turning with a druid or making small adjustment to positioning the druid turns very slow. Depending on how much you turn the animation seems to take 3 to 4 seconds longer than retail. It is not lag as the animation starts right away. With a caster spec this is not really and issue but something like cat form the auto attack will now resume until he has finished turning. This is a dps loss. Haven't done much testing yet but have noticed that sometimes after he finishes turning he doesn't continue to auto attack for a  few more seconds. Not sure if this is related to latency but I don't think it is.

**Expected behaviour**: On retail, when repositioning the turn animation finished about 3 to 4 seconds sooner. Needs to be the same here.

**Steps to reproduce the problem**:

1. Walk up to a mob
2. Start auto attack
3. Turn 180 degrees and see how slow it is and watch the auto attacks.

**Include proofs for this behaviour**I don't have any software that will record wow for me.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
